main.js

```diff
@@ -1337,10 +1337,14 @@ function sendSSH(method, cb) {
             pass: adapter.config.password
         });
 
-        ssh.exec(`echo "${adapter.config.password}"|sudo -S ${method} now`,{
+        // substitute single ' with '"'"' - 'aaa'bbb' -> 'aaa'"'"'bbb' using string concat
+        const pwd= adapter.config.password.replaceAll("'", "\'\"\'\"\'");
```

Comment:
@mcm1957 You now have an issue!! replaceAll is existing since node.js 15 ... so will break on nodejs 14! Options is to change that method or add engine tag to package.json to prevent instakls with node.js 14

Reply:
should be fixed with 2.1.11